TypeCobol.LanguageServer.Test/LSRTestHelper.cs

```diff
@@ -56,7 +56,7 @@ public static void Test(string testFolderName, LsrTestingOptions lsrTestingOptio
                 new DirectoryInfo(testWorkingDirectory + Path.DirectorySeparatorChar + "input" +
                                   Path.DirectorySeparatorChar + copyFolder).FullName.Replace(@"\", @"\\"));
             String testOptions = "";
-            testOptions += useOutline ? ",\"-ol\"" : "";
+            testOptions += useOutline ? "" : ",\"-dol\"";
```

Comment:
Line 102 in this file:
```C#
useOutline ? "-ol" : "",
```
must also be changed.

Reply:
The Idea here is that -dol is only for client option and LSR test (to keep compatibility with v1.4), command line option still to be -ol. In the future when 1.5 plugin will be available the -dol option will be removed. CobolEditEI client can use the -dol option for LSP configuration change. The server argument use -ol or not.
No no need to add -dol options as command line option.

Reply:
The "-dol" option is only used to prevent unwanted outline deactivation on a `DidChangeConfiguration` notification:
- with plugin v1.4 : "-ol" sent at startup, nothing on config change so outline is used initially and without "-dol" in config change we keep using it
- with plugin v1.5 (temporarily for compat) : "-ol" sent at startup, "-ol" or "-dol" sent in notification, so outline is used initially and kept or deactivated depending on which option is sent
- with plugin v1.5 (after migration) : "-ol" sent at startup, "-ol" sent or not in notification depending of client choice

The test code should not be modified, we can keep original code here.

```csharp
testOptions += useOutline ? ",\"-ol\"" : "";
```